src/index.js

```diff
@@ -68,6 +68,32 @@ export class Mobilitybox {
       .then(response => new MobilityboxTrip(response.data, this))
   }
 
+  /**
+   * Get a trip by its characteristics
+   * @param {object} origins_from_station MobilityboxStop or Identifier of the first station of the trip. Usually the start with: `vesputi:station:`
+   * @param {object} destination_station MobilityboxStop or Identifier of the last station of the trip. Usually the start with: `vesputi:station:`
+   * @param {object} origins_from_departure_time The departure time on the starting station as unix timestamp in milliseconds or as MobilityboxEventTime
```

Comment:
We should not switch between "origin station" and "start station". By the way: A "starting station" is a strange phrase. Better:

> The departure time of the origin station as a unix timestamp in milliseconds or a MobilityboxEventTime object